src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -1086,7 +1086,9 @@ export default {
 	padding: 0 !important;
 }
 
+// Overwrite NcListItem styles to reduce a component height
 :deep(.list-item) {
+	padding: var(--default-grid-baseline) calc(var(--default-grid-baseline) * 2);
```

Comment:
Issue on upstream?

Reply:
I'm not sure if it wasn't an intended design change for compatibility, but can upstream of course

Reply:
I'd say, if Talk needs non-standard list item design now, then something is wrong.